src/main/java/data/Event.java

```diff
@@ -100,6 +100,7 @@ public String getSaveFormat () {
                 + getEndTime();
     }
 
+    //@@author kyhjonathan-unused
```

Comment:
This can be moved to the unused folder as mentioned by Clarence. In addition, we can follow the same folder naming standard as our actual code. Nicholas has placed his unused classes in a ui folder within unused. We should similarly add our unused code belonging to a certain package into a folder with the same name within the unused folder.